Omegastation: The spring cleaning DLC

81Denton opened the pull request:
:cl: Denton
add: It's suicide HoPline prevention week at Omegastation! 
add: Navigation beacons and a brand-new Officer Beepsky unit have been installed. This means that crewmembers can now use medibots, janibots and other bots as well.
add: The atmospherics department has been outfitted with all missing gases as well as an incinerator and a hardsuit.
add: The size of the botany and xenobiology departments have been increased in order to increase productivity. The medbay O2 port has been moved so that doctors can access the table next to it.
add: RnD now has circuitry tools; medbay has stethoscopes+wrench+beaker/pill bottle boxes and primary tool storage has multitools. Enginers no longer have to share a single pair of insulated gloves either.
add: Engineers now have access to an Engi-Vend vending machine as well as missing tools like inducers. Circuitry storage has a smoke machine board and a solars crate in case the engine blows up.
tweak: Blueprints have been moved from the vault safe to secure storage in order to speed up construction projects.
fix: The AI sat transit tube now works! Turret controls can now be accessed from the outside. The first two turrets now use the proper type of gun. The telecomms air alarm will no longer report false alarms.
fix: The main hallway's firelocks now no longer all close at once.
/:cl:

[why]: While Omega has a decent size for lowpop, some areas are just too small and too sparsely equipped to allow proper gameplay (6 trays for 2 botanists? no gas mixing/incinerator for atmos techs? come on). 
This often leads to mass self spacing or HoPline suicides at roundstart; I hope this PR fixes that a bit.

[Botany before/after](https://i.imgur.com/IfV7l3M.jpg)
[Xenobiology before/after](https://i.imgur.com/IAbwjqa.jpg)
[Atmospherics before/after](https://i.imgur.com/netIA4q.jpg)
[Cryo before/after](https://i.imgur.com/Jy4wjw2.jpg)

<details><summary>Structural (click here):</summary>
<p>
- Added nav beacons with patrol paths as well as Officer Beepsky (the base model with 25 health).<br>
- Made botanics larger by cutting into the anti-breach storage next to it.<br>
- Increased xenobio's size, adding two slime pens.<br>
- Added missing gases (CO2, Plasma, N2O) as well as an incinerator and a hardsuit to atmospherics.<br>
- Moved the cryo O2 tank around so that docs can reach the table while it's wrenched down.<br>
- Split the "central primary hallway" area into multiple areas. This way, a single fire alarm doesn't shut down half the station.
</p>
</details>

<details><summary>Additions (click here):</summary>
<p>
- Circuitry storage: Smoke machine circuit board and a solars crate, since Omega doesn't come with solars by default.<br>
- Engineering: Insulated gloves (before, all engineers had to share a single pair). An engi-vend machine and a locker with electrical supplies.<br>
- Misc: Stethoscopes+medical wrench+beakers+pill bottles for medbay, multitools for tool storage. Also three arcade machines.
</p>
</details>

<details><summary>Tweaks (click here):</summary>
<p>
- Blueprints have been moved from the vault safe to Engineering secure storage. I've had lots of complaints that construction projects were neigh impossible because A) they were inside the vault safe and B) Omega had no stethoscopes either.<br>
- Replaced meson goggles in engineering with the scanner model.<br>
- <s>Reduced the AI sat SMES starting charge by 80%. The Omega minisat draws so little power that the SMES was at ~90% after a three hour round.</s> There is no power cable running from the AI sat to the station, so I reverted the AI sat SMES nerf.
</p>
</details>

<details><summary>Fixes (click here):</summary>
<p>
- The AI sat transit tube system now works. It was missing a pod and was blocked by two r-window spawners.<br>
- The telecomms air alarm now uses the correct "airalarm/server" subtype and should no longer report false alarms. The SME air alarm now also uses the correct "engine" subtype.<br>
- Fixed the position of the first AI sat turret control panel. Before, you had to stand inside the open door to access it.<br>
- Fixed the gun subtype of the first two turrets (it used to be e_gun instead of e_gun/turret).
</p>
</details>

81Denton committed:
mapmerge test

81Denton committed:
adds navigation beacons for bots

81Denton committed:
fixes wrong nav path

81Denton committed:
adds items, arcade machines, fixes server air alarm + ai sat SMES

81Denton committed:
AI sat turret panel/turret installation, misc tool positions

81Denton committed:
larger xenobio lab

81Denton committed:
xenobiology: the DLC

81Denton committed:
hydroponics expansion, fixes main hallway areas, adds smoke m. board

81Denton committed:
atmos revamp v1 before first test

81Denton committed:
atmos v2 computers, filters, incinerator

81Denton committed:
atmos v3 hidden pipes/APC capacity

81Denton committed:
atmos v4 - final tweaks, solar crate, engi electrical supplies

81Denton committed:
atmos v5 - area fixes

81Denton committed:
I want to DIE (camera tag fixes, lighting, areas)

81Denton committed:
camera range fix

81Denton committed:
removes dvars

MrDoomBringer commented:
tfw u upvote at the same time as @praisenarsie

praisenarsie commented:
!

yorii commented:
Is there a non-uplink way to open the safe in the vault yet? Please add stethoscopes to the medbay while you're at it and have an open PR.

81Denton commented:
>Is there a non-uplink way to open the safe in the vault yet?

3 or something explosions will do it

>Please add stethoscopes to the medbay while you're at it and have an open PR.

Already in the original PR heh

Cyberboss commented:
@MapDiffBot

Cyberboss commented:
@MapDiffBot

Hatfish commented:
This doesn't solve the problem of it being omegastation though.

yorii commented:
>Already in the original PR heh
Awesome! :)

arsserpentarium commented:
How about to add proper cir lab for one scientist?

newfren commented:
Finally I can stop subdividing the shit out of xenobio with glass panes and making everyone who goes in there angry at me.

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -8576,56 +8590,37 @@
-/obj/machinery/atmospherics/pipe/simple/green/visible,
-/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden{
+/obj/machinery/atmospherics/pipe/manifold/green/hidden{
+	icon_state = "manifold";
```
dv

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -8964,121 +8961,82 @@
-/obj/machinery/atmospherics/pipe/manifold/cyan/visible{
+/obj/machinery/atmospherics/components/binary/pump{
+	name = "Air to Distro";
+	on = 1
```
pump/on

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -10533,118 +10384,72 @@
-"auz" = (
-/obj/machinery/atmospherics/pipe/manifold/yellow/visible{
+/obj/machinery/atmospherics/pipe/manifold/supply/visible{
+	icon_state = "manifold";
```
dv

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -12320,133 +11972,56 @@
+"ayh" = (
+/obj/machinery/atmospherics/pipe/manifold/scrubbers/visible{
+	dir = 8;
+	name = "scrubbers pipe"
```
dv

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -14683,36 +14048,28 @@
+"aDr" = (
+/obj/structure/sign/warning/nosmoking,
 /obj/machinery/atmospherics/pipe/simple/cyan/hidden{
+	icon_state = "intact";
```
dv

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -14683,36 +14048,28 @@
+"aDs" = (
+/obj/structure/sign/warning/fire,
 /obj/machinery/atmospherics/pipe/simple/cyan/hidden{
+	icon_state = "intact";
```
dv

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -30239,6 +29495,13 @@
 /area/hallway/secondary/entry)
+"bko" = (
+/obj/machinery/atmospherics/pipe/simple/cyan/hidden{
+	icon_state = "intact";
```
dv

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -31673,11 +31028,16 @@
-	req_access_txt = "12"
+"coQ" = (
+/obj/machinery/atmospherics/components/unary/outlet_injector/on{
+	volume_rate = 50
```
dv

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -31688,11 +31048,53 @@
+"cBS" = (
+/obj/machinery/atmospherics/pipe/manifold/scrubbers/visible{
+	dir = 8;
+	name = "scrubbers pipe"
```
dv

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -31717,6 +31119,38 @@
+	},
+/turf/open/floor/plasteel/neutral/corner{
+	dir = 8;
+	heat_capacity = 1e+006
```
wut

81Denton replied:
What the? I should stop copying random objects.

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -31717,6 +31119,38 @@
+/area/engine/atmos)
+"cWR" = (
+/obj/structure/plasticflaps{
+	opacity = 1
```
intended?

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -31739,273 +31173,1727 @@
+/obj/machinery/atmospherics/components/binary/pump{
+	dir = 1;
+	name = "Oxygen to Airmix";
+	on = 1
```
pump/on

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -31739,273 +31173,1727 @@
+/obj/effect/mapping_helpers/airlock/locked,
+/obj/machinery/door/airlock/public/glass{
+	autoclose = 0;
+	frequency = 1449;
```
DV

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -31739,273 +31173,1727 @@
+/area/engine/atmos)
+"fEM" = (
+/obj/machinery/atmospherics/pipe/simple/cyan/hidden{
+	icon_state = "intact";
```
DV

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -31739,273 +31173,1727 @@
+/area/engine/atmos)
+"iqC" = (
+/obj/machinery/air_sensor{
+	frequency = 1441;
```
dv

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -31739,273 +31173,1727 @@
+"iZQ" = (
+/obj/machinery/atmospherics/components/unary/outlet_injector/on{
+	dir = 2;
+	frequency = 1441;
```
dv

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -31739,273 +31173,1727 @@
+	frequency = 1441;
+	id = "co2_in";
+	name = "co2 injector";
+	pixel_y = 1
```
intended?

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -31739,273 +31173,1727 @@
+/area/engine/atmos)
+"jDd" = (
+/obj/machinery/atmospherics/pipe/simple/cyan/hidden{
+	icon_state = "intact";
```
dv

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -31739,273 +31173,1727 @@
+/obj/machinery/atmospherics/components/binary/pump{
+	dir = 1;
+	name = "Nitrogen to Airmix";
+	on = 1
```
pump/on

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -31739,273 +31173,1727 @@
+"mgs" = (
+/obj/machinery/atmospherics/components/unary/outlet_injector/on{
+	dir = 1;
+	frequency = 1441;
```
dv

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -31739,273 +31173,1727 @@
+/area/engine/atmos)
+"mJP" = (
+/obj/machinery/igniter{
+	icon_state = "igniter0";
```
make this a /off subtype

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -31739,273 +31173,1727 @@
+/area/engine/atmos)
+"oxn" = (
+/obj/machinery/power/compressor{
+	icon_state = "compressor";
```
dv

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -31739,273 +31173,1727 @@
+"pjU" = (
+/obj/structure/grille,
+/obj/machinery/atmospherics/pipe/simple/yellow/hidden{
+	icon_state = "intact";
```
dv

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -31739,273 +31173,1727 @@
+/area/science/xenobiology)
+"pqs" = (
+/obj/machinery/power/turbine{
+	icon_state = "turbine";
```
dv

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -31739,273 +31173,1727 @@
+"pNE" = (
+/obj/structure/grille,
+/obj/machinery/atmospherics/pipe/simple/cyan/hidden{
+	icon_state = "intact";
```
dv

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -31739,273 +31173,1727 @@
+/area/engine/gravity_generator)
+"qsc" = (
+/obj/machinery/air_sensor{
+	frequency = 1441;
```
dv

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -31739,273 +31173,1727 @@
+"rVs" = (
+/obj/machinery/atmospherics/components/unary/outlet_injector/on{
+	dir = 2;
+	frequency = 1441;
```
dv

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -31739,273 +31173,1727 @@
+	frequency = 1441;
+	id = "mix_in";
+	name = "gas mix injector";
+	pixel_y = 1
```
dv?

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -31739,273 +31173,1727 @@
+	frequency = 1441;
+	id = "n2o_in";
+	name = "n2o injector";
+	pixel_y = 1
```
dv?

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -31739,273 +31173,1727 @@
+"srR" = (
+/obj/machinery/atmospherics/components/unary/outlet_injector/on{
+	dir = 2;
+	frequency = 1441;
```
dv

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -31739,273 +31173,1727 @@
+	dir = 8;
+	dwidth = 12;
+	height = 17;
+	id = "syndicate_sw";
```
uhhhhhhhhhhh

weren't these all supposed to be removed forever ago @ninjanomnom

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -34115,189 +35016,835 @@
+/area/engine/atmos)
+"uok" = (
+/obj/machinery/atmospherics/pipe/simple/cyan/hidden{
+	icon_state = "intact";
```
dv

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -34115,189 +35016,835 @@
+"vVS" = (
+/obj/machinery/atmospherics/components/unary/outlet_injector/on{
+	dir = 2;
+	frequency = 1441;
```
dv

ShizCalev commented on _maps/map_files/OmegaStation/OmegaStation.dmm:
```diff
@@ -34115,189 +35016,835 @@
+/obj/effect/mapping_helpers/airlock/locked,
+/obj/machinery/door/airlock/public/glass{
+	autoclose = 0;
+	frequency = 1449;
```
dv

81Denton committed:
removes a lot of dvars

81Denton committed:
integrates camera fixes from #36460

81Denton committed:
adds igniter/off subtype

81Denton committed:
adds light fixtures to atmos tanks

81Denton commented:
@ShizCalev That was a lot of dvars, damn. Should I remove that syndicate_sw docking port as well?

ShizCalev commented on code/game/machinery/igniter.dm:
```diff
@@ -12,6 +12,9 @@
 	resistance_flags = FIRE_PROOF
+	
+/obj/machinery/igniter/off
+	on = FALSE
```
icon_state = "igniter0"

81Denton replied:
Alright

ShizCalev reviewed (changes requested):
add

ShizCalev commented:
Not sure about the docking port, need ninja's to confirm if it's needed or not (think he was the one who removed them in the first place and all that.)

81Denton committed:
igniter icon_state

81Denton commented:
@arsserpentarium I'd like to add one, but I don't see an area I could easily convert to a circuitry lab.
I'll probably do it in a later PR since I'd have to move maintenance around a bit.

81Denton committed:
adds waste to filter pump

Cyberboss commented:
@MapDiffBot

Cyberboss commented:
@MapDiffBot

vuonojenmustaturska commented:
it's no use

F-OS commented:
i already feel proud and accomplished from this dlc and it isn't even merged

81Denton committed:
rechargers, medbay, transit pod, ai sat SMES

fixes AI sat transit tube, moves cap's plant, adds rechargers to cap+hop, adds 2 donk pockets, moves medbay oxy port, adds pill+beaker boxes

81Denton committed:
removes dvar

81Denton commented:
Alright, I fixed the AI sat transit tube system and added rechargers to the cap+HoP offices. 
I also moved the cryo O2 port around so that docs can reach the table while the canister is wrenched down.

Medbay now has the missing medical wrench, pill bottle+beaker boxes. Chef's fridge has two donk pocket boxes for long, chefless shifts.

81Denton commented:
@mapdiffbot

Cyberboss commented:
@MapDiffBot

Cyberboss commented:
@MapDiffBot

81Denton committed:
replaces SME air alarm with proper subtype
adds spare RPD, hardsuit jetpack upgrade

81Denton commented:
I was thinking about adding solars in case the SME blows up, but considering that it almost takes out half the station? That wouldn't help much.
